README.MD

```diff
@@ -18,13 +18,26 @@ usage: ocsigen-i18n-generator [options] [< input] [> output]
 
 ### ocsigen-i18n-rewriter
 
-Use the `-ppx 'ocsigen-i18n-rewriter MODULE_NAME'` where `MODULE_NAME`
-is the name of the module generated with `ocsigen-i18n-generator` (basename
-of the output file)
+The ppx supports two general forms of constructions:
+ - I18n entries in the 'default' file
+ - I18n entries in a given module
+
+You can use `ocsigen-i18n-generator` to generate multiple .eliom files
+from different TSV files. One of those will be the "default" one.
+
+Use the `-ppx 'ocsigen-i18n-rewriter DEFAULT'` where `DEFAULT`
+is the name of the module generated with `ocsigen-i18n-generator`
+you wish to use as a default for all extension expressions
+(basename of the output file)
 
 This will call the right function turning
-`[%i18n foo]` into `MODULE_NAME.foo ()`,
-`[%i18n S.bar ~x:"foo"]` into `MODULE_NAME.S.bar ~x:"foo" ()]` and so on...
+`[%i18n foo]` into `DEFAULT.foo ()`,
+`[%i18n S.bar ~x:"foo"]` into `DEFAULT.S.bar ~x:"foo" ()]` and so on...
+
+To specify a different module from the default one, you can do so using
+a expressions such as these:
+ - `[%i18n MyI18n foo]` will become `MyI18n.foo ()`
```

Comment:
Why not `MyI18n.foo` ?

Reply:
because of the `S.foo` possibility. I guess we could check the length of the dot chain and extract the first one if it's more than 2, but I really don't know how robust this is.

Reply:
Ok.